lib/elastic_record/relation.rb

```diff
@@ -70,7 +70,7 @@ def find_hits(search_hits)
       else
         ids = search_hits.to_ids
         if safe?
-          klass.where(id: ids).to_a
+          klass.where(id: ids).to_a.sort_by { |doc| ids.index { |i| i.to_s == doc.id.to_s } }
```

Comment:
Should we do something similar on line 75?

Reply:
It didn't appear that we needed to. Here is the testing I did in console around it:
```
[6] pry(#<ElasticRecord::RelationTest>)> Widget.find Widget.elastic_search.to_ids
=> [#<Widget:0x0000000107bd5350 id: 2006, warehouse_id: nil, name: nil, color: "green", price: nil, widget_part: nil>,
 #<Widget:0x0000000107bd51e8 id: 2007, warehouse_id: nil, name: nil, color: "red", price: nil, widget_part: nil>]
[7] pry(#<ElasticRecord::RelationTest>)> Widget.find Widget.elastic_search.order(color: :desc).to_ids
=> [#<Widget:0x0000000107cdff98 id: 2007, warehouse_id: nil, name: nil, color: "red", price: nil, widget_part: nil>,
 #<Widget:0x0000000107ce4098 id: 2006, warehouse_id: nil, name: nil, color: "green", price: nil, widget_part: nil>]
```
I added to the `to_a` test as well to make sure that's true.